src/components/shared/modal/modal.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component} from 'react';
+import Button from 'react-bootstrap/Button';
+import Form from 'react-bootstrap/Form';
+import Modal from 'react-bootstrap/Modal';
+import './modal.css'
+
+class ModalWindow extends Component {
```

Comment:
This is a shared component, but you've added a lot of specific logic here, you should have another component extended from this one. Something similar you have with `AboutSection`

Reply:
Done